LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
You're including code from the Go standard libraries, so I believe you would need to continue to ship with the original license:

https://golang.org/LICENSE: "Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer."

I have not checked word by word, but I think this is a 3-clause BSD license: https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22)

Reply:
Thanks, I've switched the license to the Go one. I'm awaiting advice from the licensing experts about whether anything further is needed to fix this.